### DISCUSSION TITLE
Tags: double clicking the tags opens Edit Node Tags window

### DISCUSSION THREAD

**Discussion by euu2021** [freeplane/freeplane#1908] [General]
Just to publicize a nice feature that I did not notice before, because it's a bit hidden: 

double clicking the tags in the node opens Edit Node Tags window:
![javaw_xkpoNNwGN9](https://github.com/freeplane/freeplane/assets/77707706/24681c4b-1b0e-45e2-8fb7-89905d8324b3)

**Comment by Illegit** [comment]
Thanks, useful.
